### ISSUE TITLE
Lambda layers and trainable parameters?

### ISSUE THREAD

**Issue by kesinger** [fchollet/keras#7078]
I've got a situation where I need a Conv3DTranspose, and am using the new tensorflow implementation wrapped inside a Lambda:
    def tblock(self, sinput, skip, F=32):
        u1 = Lambda(lambda x: tensorflow.layers.conv3d_transpose(x, kernel_size=3,
                              padding='SAME', strides=(2, 2, 2),
                              filters=F, activation=tensorflow.nn.relu,
                              use_bias=False))(sinput)
        out = Add()([skip,u1])
        return out

However, when I plumb layers like this into a Model, I'm seeing them show up with 0 trainable parameters, for example this .summary() putput:

    lambda_70 (Lambda)               (None, 32, 16, 16, 64 0           add_17[0][0]                     
    _____________
    add_18 (Add)                     (None, 32, 16, 16, 64 0           conv3d_6[0][0]                   
                                                                                             lambda_70[0][0]                  

I'm hoping that everything's plumbed in ok underneath, but I don't want to spend a bunch of compute time to train up a big ugly model with a couple of these layers in the middle only to find out that I should have done something differently and had trainable parameters all along.

Am I doing something clearly wrong here?
thanks!

**Comment by Danielhiversen** [comment]
Not actually answering your question, but you can use Deconvolution3D from keras_contrib:

https://github.com/farizrahman4u/keras-contrib/blob/master/keras_contrib/layers/convolutional.py#L19

**Comment by christiantinauer** [comment]
The reason why no trainable weight i showing up, is because they need to be registered (see "add_weight" in build function of layers).

Lambda may not be suitable choice. Can implement a layer yourself or contact me as i have implemented Conv3DTranspose closly related to the already existing 2D version.

**Comment by kesinger** [comment]
Thanks Daniel and Christian.  

I'll investigate add_weight, and for now I think I've got keras_contrib's Deconvolution3D working
